linkml_model/model/schema/meta.yaml

```diff
@@ -1420,6 +1420,39 @@ slots:
       - BasicSubset
       - ObjectOrientedProfile
 
+  array_info:
+    domain: slot_definition
+    range: array_info_expression
+    inherited: true
+    description: coerces the value of the slot into an array and defines the dimensions of that array
+    status: testing
+
+  dimensions_info:
+    description: definitions of each axis in the array
+    domain: array_info_expression
+    range: dimension_expression
+    multivalued: true
+    inlined: true
+    status: testing
+
+  minimum_dimensions:
+    description: minimum number of dimensions in the array
+    domain: array_info_expression
+    range: integer
+    status: testing
+
+  maximum_dimensions:
```

Comment:
separate but related question to above - it seems like ranges and exact values will/is a common pattern, thoughts on having a range syntax so we could just have a single property that can take an integer or a `1..2` specification?